lib/rubocop/config_loader.rb

```diff
@@ -58,7 +58,11 @@ def merge(base_hash, derived_hash)
 
       def base_configs(path, inherit_from)
         configs = Array(inherit_from).map do |f|
-          f = File.join(File.dirname(path), f) unless f.start_with?('/')
+          if f.include?('~')
+            f = File.expand_path(f)
+          else
+            f = File.join(File.dirname(path), f) unless f.start_with?('/')
+          end
```

Comment:
How about replacing this whole conditional with simply `f = File.expand_path(f)`?
```
irb(main):001:0> File.expand_path('~/.rubocop.yml')
=> "/home/koronen/.rubocop.yml"
irb(main):002:0> File.expand_path('Code/.rubocop.yml')
=> "/home/koronen/Code/.rubocop.yml"
irb(main):003:0> File.expand_path('/Code/.rubocop.yml')
=> "/Code/.rubocop.yml"
```